smaretski/02_HTML/task_01_table.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>  
+<HTML> 
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Прогноз погоды</TITLE> 
+    </HEAD>
+    <BODY> 
+        <center><H1>Прогноз погоды</H1></center>
```

Comment:
do not use center tag - it is weeeeery old, (use align attribute, at least)

Reply:
Fixed. Used align